FrostyModSupport/FrostyModExecutor.cs

```diff
@@ -2295,15 +2302,26 @@ private byte[] GetResourceData(string modFilename, int archiveIndex, long offset
             }
         }
 
-        private void CopyFileIfRequired(string source, string dest)
+        private void CopyFileIfRequired(string source, string dest, bool checkLength = true)
         {
             FileInfo baseFi = new FileInfo(source);
             FileInfo modFi = new FileInfo(dest);
-            if (baseFi.Exists)
+
+            if (baseFi.Exists && checkLength)
             {
-                // copy file if it doesn't exist, or recently modified
+                // copy file if it doesn't exist, recently modified, or has different file size
                 if (!modFi.Exists || (modFi.Exists && baseFi.LastWriteTimeUtc > modFi.LastWriteTimeUtc || baseFi.Length != modFi.Length))
```

Comment:
```suggestion
                if (!modFi.Exists || (modFi.Exists && baseFi.LastWriteTimeUtc > modFi.LastWriteTimeUtc || (checkLength && baseFi.Length != modFi.Length)))
```

Reply:
why do a big if else when this is all u need

Reply:
i want to make sure i dont accidently break any logic cause the if else is already setup really weirdly

Reply:
yeah the baseFi.Exist could also just be added to the if would be way cleaner